Common/src/geometry_structure.cpp

```diff
@@ -12018,13 +12019,18 @@ void CGeometry::SetResolutionTensor(void) {
       for (unsigned short iDim = 0; iDim < nDim; iDim++) {
         for (unsigned short jDim = 0; jDim < nDim; jDim++) {
           su2double temp_value = temp_tensor[iDim][jDim] / (node[iPoint]->GetnElem());
-          if (temp_value > EPS)
```

Comment:
The "positivity" test was originally included to make sure we stayed above machine precision.  Otherwise, I would get funny numbers (that were still technically correct) like 1.0000000000000001 or 0.99999999999999999.

So this change is fine by me.

Reply:
Only adding the positive contributions is incorrect.  That is the reason for the change.  The eigenvalues of the resolution tensor have to be positive, but the entries do not, so if you only add the positive entries here you'll get the wrong answer (on a general mesh... the result on an orthogonal mesh should be ok).  If you want to avoid adding values that are order machine precision (e.g., to avoid the funny numbers you mention) you could test that abs(temp_value) > EPS, but I don't see the point.  That is what the cryptic comment was intended to convey.